src/system/rqgCalculations.ts

```diff
@@ -5,20 +5,38 @@ type LookupTableEntry<T> = {
 };
 
 export class RqgCalculations {
-  public static dexSR(dex: number): number | undefined {
+  public static dexSR(dex: number | undefined): number | undefined {
+    if (dex == null) {
+      return undefined;
+    }
     return this.lookup<number | undefined>(dex, dexSrTable);
   }
 
-  public static sizSR(siz: number): number | undefined {
+  public static sizSR(siz: number | undefined): number | undefined {
+    if (siz == null) {
+      return undefined;
+    }
     return this.lookup<number | undefined>(siz, sizSrTable);
   }
 
-  public static hitPoints(con: number, siz: number, pow: number): number {
-    return con + RqgCalculations.linearMod(siz) / 5 + RqgCalculations.flattenedMod(pow) / 5;
+  public static hitPoints(
+    con: number | undefined,
+    siz: number | undefined,
+    pow: number | undefined
+  ): number | undefined {
+    let hp =
+      con != null && siz != null && pow != null
+        ? con + RqgCalculations.linearMod(siz) / 5 + RqgCalculations.flattenedMod(pow) / 5
+        : undefined;
+    if (hp && hp < 3) {
+      // HP floor since 1 or 2 is unconscious.
+      hp = 3;
+    }
+    return hp;
   }
 
-  public static damageBonus(str: number, siz: number): string {
-    if (str == null && siz == null) {
+  public static damageBonus(str: number | undefined, siz: number | undefined): string {
+    if (str == null || siz == null) {
       return "0";
```

Comment:
I think this is getting into the question of what it means to not have either STR or SIZ.  There's a difference between 0 and not having it at all.   I think not having either STR or SIZ probably means you don't have a damage bonus because physical damage isn't a thing you can do anyway.  Having zero in either STR or SIZ or both means you have a physical form but are either incapacitated or permanently paralyzed or I suppose zero SIZ is just super tiny like a tardigrade or something.  I think there's a chart for the SIZ that might clear that up.

Reply:
Thinking this through again with the code as it is right now:

STR 18, SIZ 18 = 1d6 Damage Bonus (makes sense)
no STR, SIZ 32 = 1d4 Damage Bonus (ok, I could buy this in that a SIZ 32 object could fall on you, but it should probably be "Does not inflict physical damage")
STR 32, no SIZ = 1d4 Damage Bonus (I really think this should be "Does not inflict physical damage".  It doesn't matter how strong you are if you don't have physical form)

I tried it with the change you're suggesting and it's the same exact result.

The ones with no STR or no SIZ should probably not be "0" (zero), but should be "does not inflict physical damage" or something like that.